### PR TITLE
Add new ACERetro backend service

### PR DESCRIPTION
**Pull request by KastanDay**
* Inputs stored in Minio: `/{job_id}/in/input.json`. Bucket name: `aceretro`
* Outputs stored in Minio: `/{job_id}/out/output.json`. Bucket name: `aceretro`

### CODE REVIEW

File: app/routers/files.py [hunk 1]
```diff
@@ -54,6 +55,10 @@ async def get_results(bucket_name: str, job_id: str, service: MinIOService = Dep
         print("Getting CHEMSCRAPER job result")
         return await ChemScraperService.resultPostProcess(bucket_name, job_id, service, db)
     
+    if bucket_name == JobType.ACERETRO:
+        print("Getting CHEMSCRAPER job result")
```
Comment (matthewberry) [inline app/routers/files.py:59]: this message should probably say "ACERETRO"

File: app/routers/job.py [hunk 2]
```diff
@@ -113,14 +153,28 @@ async def create_job(
                 'name': 'SOMN_PROJECT_DIR',
                 'value': somn_project_dir
             }]
-
+
```
Comment (matthewberry) [inline app/routers/job.py:156]: do we still need this TODO comment?

File: app/cfg/local.config.yaml [hunk 3]
```diff
@@ -140,10 +140,10 @@ data:
...
           subPath: uws/jobs/somn/JOB_ID/out
     minio:
-      accessKey: ""
-      apiBaseUrl: minioapi.mmli.fastapi.localhost
-      secretKey: ""
-      server: mmli-backend-minio.mmli.svc.cluster.local:9000
+      server: "minio.mmli.fastapi.staging.mmli1.ncsa.illinois.edu"
+      apiBaseUrl: "minioapi.mmli.fastapi.staging.mmli1.ncsa.illinois.edu"
+      accessKey: "9fc1267b77741fbbf89a4b93cb95f9be"
+      secretKey: "b2f9a572c7d0c822ffd2f08224529ee7"
```
Comment (bodom0015) [inline app/cfg/local.config.yaml:146]: @KastanDay we don't want to check secrets into git in a public repo, and certainly not in production .. please remove this section.

**Review by bodom0015 — CHANGES_REQUESTED** [review]
Please rebase to remove your last commit :pray:

File: app/routers/files.py (hunk 1 above)
Comment (KastanDay) [reply app/routers/files.py:59]: Done, thank you!

File: app/routers/job.py (hunk 2 above)
Comment (KastanDay) [reply app/routers/job.py:161]: No, that has been completed. Thank you!

File: chart/values.staging.yaml [hunk 4]
```diff
@@ -57,6 +57,7 @@ config:
   somn_frontend_url: "https://somn.frontend.staging.mmli1.ncsa.illinois.edu"
   clean_frontend_url: "https://clean.frontend.staging.mmli1.ncsa.illinois.edu"
   molli_frontend_url: "https://molli.frontend.staging.mmli1.ncsa.illinois.edu"
+  aceretro_frontend_url: "https://aceretro.frontend.staging.mmli1.ncsa.illinois.edu"
```
Comment (ckouder) [inline chart/values.staging.yaml:60]: update staging.yaml for frontend_url

File: chart/values.prod.yaml [hunk 5]
```diff
@@ -60,6 +60,7 @@ config:
   somn_frontend_url: "https://somn.platform.moleculemaker.org"
   clean_frontend_url: "https://clean.platform.moleculemaker.org"
   molli_frontend_url: "https://molli.platform.moleculemaker.org"
+  aceretro_frontend_url: "https://aceretro.platform.moleculemaker.org"
```
Comment (ckouder) [inline chart/values.prod.yaml:63]: update prod.yaml for frontend_url

**Review by bodom0015 — APPROVED** [review]
LGTM :tada: Thank you for all your work on ACERetro!

**Pull request merged**